tests/unit/date-range-input.spec.ts

```diff
@@ -334,6 +334,25 @@ describe('Date range input', () => {
     });
   });
 
+  describe('with relative date disabled', () => {
+    beforeEach(() => {
+      createWrapper({
+        availableVariables: SAMPLE_VARIABLES,
+        variableDelimiters: { start: '{{', end: '}}' },
+        enableRelativeDate: false,
+      });
+    });
+    it('should have no tabs', () => {
+      expect(wrapper.find('Tabs-stub').exists()).toBe(false);
+    });
+    it('should always use "Fixed" as selected tab', () => {
+      expect((wrapper.vm as any).selectedTab).toBe('Fixed');
+    });
```

Comment:
May be we should test what we expect to be displayed relative to this internal state ? Here the `RangeCalendar` ?

Reply:
:+1:

Reply:
https://github.com/ToucanToco/weaverbird/pull/1095/commits/a628048ee9b6e6f5aeb05bc451fdb2635162ba9b